### PR TITLE
for now don't pass in maxFee if all fees is selected

### PR DESCRIPTION
**Pull request by bthaile**
Just a stop gap so that markets with 0% fees will show up when user select all fees. 

Still an issue in sdk returning markets with 0% fees.

### CODE REVIEW

**Review by ghost — COMMENTED** [review]
This pull request has been automatically deployed to [FeaturePeek](https://featurepeek.com). 🎯

 This link will always point to this PR's latest changes: https://peek.run/654h2pp

**Comment by andrewdamelio** [comment]
@bthaile seeing errors on featurepeak

File: packages/augur-ui/src/modules/markets/actions/load-markets.ts [hunk 1]
```diff
@@ -128,12 +128,16 @@ export const loadMarketsByFilter = (
...
+    maxFee:
+      filterOptions.maxFee === MAX_FEE_100_PERCENT ? '' : filterOptions.maxFee,
     includeInvalidMarkets: filterOptions.includeInvalidMarkets,
     limit: filterOptions.limit,
     offset: paginationOffset * filterOptions.limit,
     reportingStates,
-    maxLiquiditySpread: filterOptions.maxLiquiditySpread as Getters.Markets.MaxLiquiditySpread,
+    maxLiquiditySpread:
+      filterOptions.maxLiquiditySpread === MAX_SPREAD_ALL_SPREADS
+        ? ''
```
Comment (bconfortin) [inline packages/augur-ui/src/modules/markets/actions/load-markets.ts]: I don't think you can pass an empty string as a value, according to FeaturePeek.
`maxLiquiditySpread: "0" | "10" | "15" | "20" | "100"`

```javascript
ww-connector.ts:123 Uncaught (in promise) Error: Invalid request object: Invalid value "" supplied to : (({ universe: string } & Partial<{ creator: string, search: string, reportingStates: Array<string>, designatedReporter: string, maxFee: string, maxEndTime: number, maxLiquiditySpread: "0" | "10" | "15" | "20" | "100", includeInvalidMarkets: boolean, categories: Array<string>, sortBy: "marketOI" | "liquidity" | "volume" | "timestamp" | "endTime" | "lastTradedTimestamp" | "disputeRound" | "totalRepStakedInMarket", userPortfolioAddress: string }>) & Partial<{ sortBy: string, isSortDescending: boolean, limit: number, offset: number }>)/0: ({ universe: string } & Partial<{ creator: string, search: string, reportingStates: Array<string>, designatedReporter: string, maxFee: string, maxEndTime: number, maxLiquiditySpread: "0" | "10" | "15" | "20" | "100", includeInvalidMarkets: boolean, categories: Array<string>, sortBy: "marketOI" | "liquidity" | "volume" | "timestamp" | "endTime" | "lastTradedTimestamp" | "disputeRound" | "totalRepStakedInMarket", userPortfolioAddress: string }>)/1: Partial<{ creator: string, search: string, reportingStates: Array<string>, designatedReporter: string, maxFee: string, maxEndTime: number, maxLiquiditySpread: "0" | "10" | "15" | "20" | "100", includeInvalidMarkets: boolean, categories: Array<string>, sortBy: "marketOI" | "liquidity" | "volume" | "timestamp" | "endTime" | "lastTradedTimestamp" | "disputeRound" | "totalRepStakedInMarket", userPortfolioAddress: string }>/maxLiquiditySpread: "0" | "10" | "15" | "20" | "100"
    at eval (ww-connector.ts:123)
    at Array.forEach (<anonymous>)
    at Worker.worker.onmessage (ww-connector.ts:121)
```

**Review by bconfortin — CHANGES_REQUESTED** [review]

File: packages/augur-ui/src/modules/markets/actions/load-markets.ts (hunk 1 above)
Comment (bthaile) [reply packages/augur-ui/src/modules/markets/actions/load-markets.ts]: I can delete the property instead of passing empty

Comment (bconfortin) [reply packages/augur-ui/src/modules/markets/actions/load-markets.ts]: Didn't work for maxLiquiditySpread, at least on FeaturePeek

Isn't 100 the maxLiquiditySpread?
Shouldn't setting the value to 100 work?

Comment (bthaile) [reply packages/augur-ui/src/modules/markets/actions/load-markets.ts]: if 100 we don't send. recent decision

**Review by bconfortin — APPROVED** [review]

**Pull request merged**